src/config.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  buildAgentConfig,
+  ChainMap,
+  ChainName,
+  CoreConfig,
+  defaultMultisigIsmConfigs,
+  GasOracleContractType,
+  HyperlaneAddresses,
+  HyperlaneAgentAddresses,
+  MultiProvider,
+  MultisigIsmConfig,
+  multisigIsmVerificationCost,
+  objFilter,
+  objMerge,
+  OverheadIgpConfig,
+} from '@hyperlane-xyz/sdk';
+import { hyperlaneEnvironments } from '@hyperlane-xyz/sdk/dist/consts/environments';
+import { types, utils } from '@hyperlane-xyz/utils';
+import { ethers } from 'ethers';
+import artifactAddresses from '../artifacts/addresses.json';
+import { chains } from '../config/chains';
+import { multisigIsmConfig } from '../config/multisig_ism';
+import { readJSON } from './json';
+
+export function getMultiProvider() {
+  const multiProvider = new MultiProvider();
+  for (const metadata of Object.values(chains)) {
+    multiProvider.addChain(metadata);
+  }
+  return multiProvider;
+}
+
+export function assertBytes32(value: string): string {
```

Comment:
@jmrossy introduced a library called zod for shape validation in the SDK

Reply:
I'm sure it's better but I'd really like to ship this